pkg/porter/explain.go

```diff
@@ -379,3 +401,30 @@ func (p *Porter) printActionsExplainTable(bun *PrintableBundle) error {
 		}
 	return printer.PrintTable(p.Out, bun.Actions, printActionRow, "Name", "Description", "Modifies Installation", "Stateless")
 }
+
+// Dependencies
+func (p *Porter) printDependenciesExplainBlock(bun *PrintableBundle) error {
+	if len(bun.Dependencies) > 0 {
+		fmt.Fprintln(p.Out, "Dependencies:")
+		err := p.printDependenciesExplainTable(bun)
+		if err != nil {
+			return errors.Wrap(err, "unable to print dependencies table")
+		}
+	} else {
+		fmt.Fprintln(p.Out, "No dependencies defined")
+	}
+	fmt.Fprintln(p.Out, "") // force a blank line after this block
+	return nil
+}
+
+func (p *Porter) printDependenciesExplainTable(bun *PrintableBundle) error {
+	printOutputRow :=
```

Comment:
Can `printOutputRow` be updated to `printDependencyRow` (or similar)?

Reply:
You are right totally missed it